### ISSUE TITLE
Proposal

### ISSUE THREAD

**Issue by farid-temuri** [zabulonov/ConfirmationService#2]
I would like to discuss harassment this repository projects to other members of our community. The source is HUGE COCK ENERY, PROJECTED TO US. The biggest to be exact

**Comment by zabulonov** [comment]
Dear Farid,

Thank you for your issue on this topic, this is a very interesting problem that requires **DEEP** study. The author of this repository is already aware of this problem and is working to fix it.

For users who want to fork this repo and who are hindered by this **HUGE ENERGY**, I recommend that you start improving your fork :)

I advise you to use technologies such as: JELQING, PUMP, and tablets (consult your doctor, we do not use such libraries). It is also important to create a roadmap and keep logs (write down your progress on a piece of paper).

Also, don’t be upset if your fork is very small, I have attached a picture below with the average fork size by country.

![image_2022_04_22T17_09_03_491Z](https://github.com/zabulonov/ConfirmationService/assets/83907630/3be25b87-36c8-496b-bec1-02c7779494b0)

Thank you for your issue, I hope I was able to help you somehow and everything will be fine in your life.

All the best!
